README.md

```diff
@@ -1,63 +1,72 @@
 # 🌱🎭🌍
 
-# 🏗 Scaffold-ETH - 🎟 Simple NFT Example
+# Time Guarden
```

Comment:
Typo on "Guarden" or intended wordplay?

Reply:
There is only intended wordplay with this one.